published/003-2016-Jan-12.md

```diff
@@ -0,0 +1,27 @@
+# IPFS Weekly #3
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+Here are some of the highlights for the [January 12th](//github.com/ipfs/pm/issues/79) Sprint:
+
+## Updates
+
+* [go-ipfs](//github.com/ipfs/go-ipfs) 0.3.11 is shipped! Also, we merged 0.4.0 into master. This is huge news that we've been waiting for for a long time. Also, [@jbenet](//github.com/jbenet), [@Dignifiedquire](//github.com/Dignifiedquire) and others [fixed appveyor](//github.com/ipfs/go-ipfs/pull/2137). The latest webui by [@Dignifiedquire](//github.com/Dignifiedquire) was also pushed in 0.3.11.
```

Comment:
`This is huge news that we've been waiting for for a long time.` This sentence sounds strange :/ (sorry for not being more helpful)